tracetesting/run.bash

```diff
@@ -23,15 +23,15 @@ echo "TRACETEST_MAIN_ENDPOINT: $TRACETEST_MAIN_ENDPOINT"
 echo "TRACETEST_TARGET_ENDPOINT: $TRACETEST_TARGET_ENDPOINT"
 echo "DEMO_APP_URL: $DEMO_APP_URL"
 
-tee config.main.yml << END
+cat << EOF > config.main.yml
```

Comment:
`tee` outputs to stdout in addition of appending to the file. Using `cat` does not, so the ouput looks cleaner